docs/event-mesh/10-01-troubleshooting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Troubleshooting
+---
+
+## Events do not reach the Namespace with active Eventing
+
+The upgrade of the Event Sources Controller Manager triggers the new Pod to update the KService for the HTTPSource. As a result of changes introduced to the KService, the Knative Serving Controller (KSC) creates a new Revision. The KSC does not pick up this new Revision and marks the KService as `Not ready`, still looking for the old Revision.
+To fix this issue, delete the existing KService. It will be recreated automatically, pointing to the right Revision.
+
+Follow these steps:
+
+1. Check the status of the HTTP Source:
+
+    ```bash
+    $ kubectl get httpsources.sources.kyma-project.io -n kyma-integration
+     NAME          READY   REASON
```

Comment:
There is an alignment issue with `READY`. I think there is one space extra before `READY`